frontend/src/components/common/input-password/input-password.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, FC } from 'react';
+import { Input } from 'components/common/common';
+import {
+  FormControl,
+  FormControlErrors,
+  FormControlPath,
+} from 'common/types/types';
+import { InputType } from 'common/enums/enums';
+import styles from './input-password.module.scss';
+
+type Props = {
+  label: string;
+  name: FormControlPath;
+  control: FormControl;
+  errors: FormControlErrors;
+  placeholder?: string;
+};
+
+const PasswordInput: FC<Props> = ({
+  label,
+  name,
+  control,
+  errors,
+  placeholder = '',
+}) => {
+  const [isShowPassword, setIsShowPassword] = useState<boolean>(false);
+
+  const handleShowToggle = (): void => {
+    setIsShowPassword(!isShowPassword);
+  };
+
+  return (
+    <div>
```

Comment:
```suggestion
    <div className={styles.wrapper}>
```
make this a `relative` and add an `absolute` to the button